hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/bucket/CreateBucketHandler.java

```diff
@@ -62,7 +62,7 @@ public class CreateBucketHandler extends BucketHandler {
   enum AllowedBucketLayouts {
```

Comment:
We can simplify the Enum. The constructor, toString override, and member variable were only required for the case where we wanted to allow an empty string to be passed as a value to the layout option.
Since now we are disallowing the empty string as a value - we can remove reduce the complexity here.
```suggestion
  enum AllowedBucketLayouts { FILE_SYSTEM_OPTIMIZED, OBJECT_STORE, LEGACY }
```

Reply:
Thank you for the review, I have made the changes!